spec/system/page_objects/components/sidebar.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Components
+    class Sidebar < PageObjects::Components::Base
+      def has_category_section_link?(category)
+        page.has_link?(category.name)
```

Comment:
There is nothing specific to sidebar here...if a category linked by name shows literally anywhere on the page this will pass. It should be scoped to the sidebar link class e.g. `page.has_css(".sidebar-section-link", text: category.name)` (or whatever the link CSS is)

Reply:
O good catch. I've included `class: "sidebar-section-link"` to `has_link?`